source_code/aux_mcu_v4/src/fido2/ctap.h

```diff
@@ -218,21 +217,7 @@ typedef struct
 
 typedef struct
 {
-    uint8_t saltLen;
-    uint8_t saltEnc[64];
-    uint8_t saltAuth[32];
-    COSE_key keyAgreement;
-    struct Credential * credential;
-} CTAP_hmac_secret;
```

Comment:
This is an optional extension in CTAP:
This extension is used by the platform to retrieve a symmetric secret from the authenticator when it needs to encrypt or decrypt data using that symmetric secret. This symmetric secret is scoped to a credential. The authenticator and the platform each only have the part of the complete secret to prevent offline attacks. This extension can be used to maintain different secrets on different machines.